src/gtk/window.cpp

```diff
@@ -5399,6 +5399,67 @@ void wxWindowGTK::Lower()
     }
 }
 
+// ----------------------------------------------------------------------------
+// Cursor stuff
+// ----------------------------------------------------------------------------
+
+// Return true if there is a global (usually busy) cursor overriding all the
+// other ones.
+static bool wxIsUsingGlobalCursor(wxWindowGTK* w)
+{
+    if (g_globalCursor.IsOk())
+        return true;
+
+    if (wxIsBusy())
+    {
+        wxWindow* win = wxGetTopLevelParent(w);
+        if (win && win->m_widget && !gtk_window_get_modal(GTK_WINDOW(win->m_widget)))
+            return true;
+    }
+
+    return false;
+}
+
+wxArrayGdkWindows wxWindowGTK::GTKSetCursorForAllWindows(GdkCursor* cursor)
+{
+    wxArrayGdkWindows changed;
```

Comment:
Isn't `changed` the same as `windows`? Doesn't seem like you need to make a copy.

Reply:
`windows` may contain null elements.